apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -31,6 +38,76 @@ function should_show_coming_soon_page() {
 	return apply_filters( 'a8c_show_coming_soon_page', $should_show );
 }
 
+/**
+ * Gets share code from URL or Cookie.
+ *
+ * @return string
+ */
+function get_share_code() {
+	if ( isset( $_GET['share'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
What do you think about using a more descriptive query param like `?site_preview`? We could use the same for the cookie name.

Reply:
I think `share_code` would be more descriptive for the parameter, and Site Preview is a codename of the whole feature we are adding. Thus I used `share_code` for a cookie name and shortened it for the URL param as shorter param seems nicer.